app/views/requests/_request_row.html.erb

```diff
@@ -1,9 +1,15 @@
 <tr>
   <td class="date"><%= request_row.created_at.strftime("%m/%d/%Y") %></td>
-  <td><%= request_row.partner.name %> </td>
-  <td>
-    <%= render partial: "status", locals: {status: request_row.status} %>
-  </td>
+  <td><%= request_row.partner.name %></td>
+  <% if request_row.partner.distribution_limit > 0 && request_row.request_items.count > request_row.partner.distribution_limit %>
```

Comment:
Looks like you're counting the number of request item categories rather that the quantity of diapers received.
For example if the request is for 500 size 1, and 500 size 2 - We want the item count and display to be 1000 and we're currently showing 2.  We calculate this elsewhere, you should look at the total items column on the distributions index page.